container_pipeline/lib/default_settings.py

```diff
@@ -9,8 +9,18 @@
 LOG_LEVEL = os.environ.get('LOG_LEVEL') or 'DEBUG'
 
 LOG_PATH = '/srv/pipeline-logs/cccp.log'
+
 SERVICE_LOGFILE = "service_debug.log"
 
+LINTER_RESULT_FILE = "linter_results.txt"
+
```

Comment:
New lines are not required after each line. You may want to group multiple lines into logical paragraph and have an empty line between paragraphs.